src/pages/projects-list/ProjectsList.tsx

```diff
@@ -110,6 +109,7 @@ function getConfig(history: History, currentUser: CurrentUser) {
             icon: "assignment",
             text: i18n.t("Add Target Values"),
             multiple: false,
+            onClick: () => history.push(generateUrl("targetValues")),
```

Comment:
To pass the id of the project (see my comment below) have a look at what we are currently doing in the Edit action